Documentation/AEPEdgeIdentity.md

```diff
@@ -0,0 +1,90 @@
+# Getting Started
+
+# Test App
+## GAID Background
+In Android, users are opted-in to ad ID tracking by default. They can choose to opt out of tracking in Android Settings at the device level.
+Developers using ad ID should get ad ID from the API each time it is used, as permissions for ad tracking and/or the value of the ID itself may be changed at any time.
+- In older versions of Android, the ad ID opt-in/out is a toggle, where the existing ad ID value remains unchanged.
```

Comment:
will be good to add a screenshot

Reply:
Added assets folder with screenshots, and img links with standardized width